pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -632,6 +634,25 @@ func emitSchedule(
 	return nil
 }
 
+func checkScheduleAlreadyExists(
+	ctx context.Context, p sql.PlanHookState, scheduleLabel string,
+) (bool, error) {
+
+	// the last
+	rows, err := p.ExecCfg().InternalExecutor.QueryRowEx(ctx, "check_sched", nil, sessiondata.InternalExecutorOverride{User: security.RootUserName()},
+		fmt.Sprintf("SELECT COUNT(schedule_name) FROM %s WHERE schedule_name = %s",
```

Comment:
You probalby don't need to return count at all.
Just select e.g. `schedule_id WHERE schedule_name=`.
If `rows == nil` -- it means it doesn't exist

Reply:
(you can also add LIMIT 1)

Reply:
just added LIMIT 1. done.